server/events/vcs/gh_app_creds_rotator.go

```diff
@@ -18,19 +18,22 @@ type githubAppTokenRotator struct {
 	log               logging.SimpleLogging
 	githubCredentials GithubCredentials
 	githubHostname    string
+	gitUser           string
 	homeDirPath       string
 }
 
 func NewGithubAppTokenRotator(
 	log logging.SimpleLogging,
 	githubCredentials GithubCredentials,
 	githubHostname string,
+	gitUser string,
```

Comment:
This function name and type need renaming to `githubTokenRotator` now that you have made it not specific to the app.